packages/rest/src/rest.server.ts

```diff
@@ -197,6 +197,11 @@ export class RestServer extends Context implements Server, HttpServerLike {
       return Promise.resolve();
     }
 
+    // Any item down the sequence chain can override the statusCode if needed
+    if (request.method === 'POST') {
```

Comment:
This is too hacky. Not all `POST` is for `create`. Can we allow/infer the statusCode from https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#responses-object?

Reply:
There are a few potential options for controller methods to express status code:
1. Direct access to `Response` object
2. Decorate the method to customize the default status code
3. Return a `HttpResponse` object such as `{statusCode: 201, headers: {}, body: result}`

Reply:
We have issues to track the stories of providing a way for users to set the statusCode and appropriate response headers. 

I personally like option 3 -- but for a `POST` request I think we should automatically be defaulting to 201 if a different statusCode is not provided.

Reply:
@virkt25 After I did my PoC in https://github.com/strongloop/loopback-next/pull/1260, I tend to agree with your workaround here as the quickest fix....or allowing more time, we can do 3.

Miroslav brings an important thing: when a response returns `201`, it also includes a header with the URI of created item. 

> but for a POST request I think we should automatically be defaulting to 201 if a different statusCode is not provided.

Then the user need to provide `header` in `result` by default as well.